src/main/java/no/ccat/service/RDFToModelTransformer.java

```diff
@@ -441,4 +455,15 @@ private LocalDate extractDateFromResource(Resource resource, Property property)
         }
         return null;
     }
+
+    private boolean randomizeForTest(){
+        boolean local = false;
+
+        for (String profile :  env.getActiveProfiles()){
+            if(profile.equals("test") || profile.equals("dev-with-harvester") ){
+                local = true;
+            }
+        }
+        return local;
+    }
```

Comment:
Flytt til test

Reply:
Synes fortsatt at denne dato-manipuleringen blir feil. Er bedre å fylle databasen med data du har full kontroll på

Reply:
Forsåvidt enig, er jo en fulfix. Gjorde det sånn fordi det var en rask måte å få data å teste på. first harvested er jo ikke kontrolerbar i utgangspunktet, så eneste alternativet sånn som jeg skjønner det er å oppdatere det i elasticsearch etter at programmet har kjørt, noe som kjennes ut som å åpne en can of worms.

Reply:
Er enig med det @azaxarov sa på standup. Bedre å fjerne de respektive testene enn å legge til testkode her

Reply:
Ja, skal fikse det nå :)